README.md

```diff
@@ -145,13 +146,28 @@ Facilitators:
 - Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
 - Zach Corleissen ([@zacharysarah](https://github.com/zacharysarah)), Linux Foundation
 
-### GitHub Management
+### Inclusiveness Team
 
-Status: [Proposed](https://github.com/cncf/tag-contributor-strategy/issues/5)
+Facilitators
 
-Facilitators:
+- Melania De Angelis ([@melaniadeangelis](https://github.com/melaniadeangelis))
+- Sara Trapetti ([@SarahTrap](https://github.com/SarahTrap))
+- Jade Applegate ([@jadeapplegate](https://github.com/jadeapplegate))
 
```

Comment:
We should probably be consistent about listing affiliations and add the affiliations for these new members.

Reply:
Sure -- except that I have no idea who their employers are.  Jade/Sara/Melania?

Reply:
tracked them down